### DISCUSSION TITLE
How to see which keymap trigger what action?

### DISCUSSION THREAD

**Discussion by rjshrjndrn** [LazyVim/LazyVim#2127] [Q&A] [answered]
For example,

`<c-t>` in  
`:Telescope oldfiles`

open trouble, rather than opening files in another tab

`<esc><esc>k` will move current line 1 line above. 

You'll get some surprises while editing files, especially if you're fast with keybindings.

I tried `:maps` and there is nothing related to these there.

**Comment by dpetka2001** [comment]
These mappings are internal to Telescope ~~and have nothing to do with LazyVim~~. If you press `<C-/>` in insert mode in Telescope or `?` in normal mode in Telescope it will show you the available mappings they have documented. You can also [here](https://github.com/LazyVim/LazyVim/blob/879e29504d43e9f178d967ecc34d482f902e5a91/lua/lazyvim/plugins/editor.lua#L227-L240) in the repo what mappings Lazyvim uses/changes for Telescope.

**Reply by rjshrjndrn** [reply]
yeah, lazy is creating the keymap
https://github.com/LazyVim/LazyVim/blob/879e29504d43e9f178d967ecc34d482f902e5a91/lua/lazyvim/plugins/editor.lua#L229

**Reply by rjshrjndrn** [reply]
@dpetka2001 is there any way to disable only this key map?

**Reply by dpetka2001** [reply]
Something like the following should work
```lua
  {
    "nvim-telescope/telescope.nvim",
    opts = {
      defaults = {
        mappings = {
          i = {
            ["<c-t>"] = false,
          },
        },
      },
    },
  },
```

**Reply by rjshrjndrn** [reply, marked as answer]
thanks @dpetka2001 

Finally I've settled with this
```lua
return {
  "nvim-telescope/telescope.nvim",
  keys = {
    -- disable the keymap to grep files
    { "<leader>/", false },
  },
  opts = {
    defaults = {
      mappings = {
        i = {
          ["<c-t>"] = require("telescope.actions").select_tab,
        },
      },
    },
  },
}
```

**Reply by dpetka2001** [reply]
Oh you wanted the default Telescope action to `<c-t>`. I thought you wanted to disable it completely. I misunderstood. By the way I believe you can just use `actions.select_tab` with the builtin Telescope actions. I hope I'm not wrong with that.
